darts/timeseries.py

```diff
@@ -21,9 +21,9 @@
     - Have a monotonically increasing time index, without holes (without missing dates)
     - Contain numeric types only
     - Have distinct components/columns names
-    - Have a well defined frequency (
-    `date offset aliases <https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases>`_
-    for ``DateTimeIndex``, and step size for ``RangeIndex``)
+    - Have a well defined frequency (`date offset aliases
+    <https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#offset-aliases>`_
+    for ``DateTimeIndex``, or step size for ``RangeIndex``)
```

Comment:
the formatting is still wrong in the compiled documention, need to find a solution, maybe moving this bullet point to the bottom